gulp/html/copy.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var gulp = require('gulp'),
+    htmlTask = require('./default');
+
+var taskName = 'html:copy',
+	taskConfig = {
+		src: htmlTask.taskConfig.src,
```

Comment:
This will skip some partials, e.g. https://github.com/unic/estatico/blob/develop/gulp/html/default.js#L18